contrib/gce/configure.sh

```diff
@@ -104,6 +104,15 @@ if [ -f "${CONTAINERD_HOME}/${CONTAINERD_ENV_METADATA}" ]; then
   source "${CONTAINERD_HOME}/${CONTAINERD_ENV_METADATA}"
 fi
 
+# GCS_BUCKET_TOKEN_METADATA is the metadata key for the GCS bucket token
```

Comment:
Is it okay to keep `set -o xtrace` on the top of this file? I'm fine removing that to be honest.

Reply:
@kzys Have updated the changes so that tracing is disabled when dealing with tokens